frontend/ui/src/pages/Inbox/MessageInput/index.tsx

```diff
@@ -71,13 +75,20 @@ const MessageInput = (props: MessageInputProps & ConnectedProps<typeof connector
   const sendMessage = () => {
     if (selectedTemplate) {
       setSelectedTemplate(null);
-      props
-        .sendMessages({conversationId: conversation.id, message: selectedTemplate.message.content})
-        .then(() => setInput(''));
+      sendMessages({conversationId: conversation.id, message: selectedTemplate.message.content}).then(() =>
+        setInput('')
+      );
+      return;
+    }
+    if (selectedSuggestedReply) {
+      setSelectedSuggestedReply(null);
+      sendMessages({conversationId: conversation.id, message: selectedSuggestedReply.message.content}).then(() =>
+        setInput('')
+      );
       return;
     }
 
-    props.sendMessages(getTextMessagePayload(source, conversation.id, input)).then(() => setInput(''));
+    sendMessages(getTextMessagePayload(source, conversation.id, input)).then(() => setInput(''));
```

Comment:
I think we could reduce some code in this method. Just use `sendMessages` once at the end, so the dynamic thing is the payload